### PR TITLE
Fixes #104 - Adds route for triage bugs list

### LINKED ISSUE #104: Move the backend webcompat.com triage dashboard code to here
This will 
* simplify the code on webcompat.com
* remove the CORS issues identified in https://github.com/webcompat/webcompat.com/issues/2866

It is the server side of https://github.com/webcompat/webcompat-metrics-client/issues/172

### PR DESCRIPTION
**Pull request by karlcow**
This will probably solve the issues that @magsout is meeting currently with the client part and it will help us to clean up a bit webcompat.com

### CODE REVIEW

File: ochazuke/__init__.py [hunk 1]
```diff
@@ -196,6 +196,20 @@ def sitewait_data():
         response.headers.add('Vary', 'Origin')
         return response
 
+    @app.route('/data/triage-bugs')
+    def triage_bugs():
+        """Returns the list of issues which are currently in triage."""
+        url = 'https://api.github.com/repos/webcompat/webcompat-tests/issues?sort=created&per_page=100&direction=asc&milestone=1'  # noqa
```
[the remark this answers is not in the document]
Comment (karlcow) [reply ochazuke/__init__.py]: doh! that's what reviews are for.  ❤️ Thanks for spotting it. 
typo typo typo. :) I will fix it.

File: ochazuke/__init__.py [hunk 2]
```diff
@@ -196,6 +196,20 @@ def sitewait_data():
         response.headers.add('Vary', 'Origin')
         return response
 
+    @app.route('/data/triage-bugs')
```
[the remark this answers is not in the document]
Comment (karlcow) [reply ochazuke/__init__.py]: nope. mistake from me.

File: ochazuke/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (karlcow) [reply ochazuke/__init__.py]: Ok this is done. Thanks.

**Comment by karlcow** [comment]
@magsout This is deployed.

**Comment by magsout** [comment]
@karlcow awesome, thanks a lot. i'm coding the new dashboard.. i'm late ..

**Pull request merged**